packages/api/src/functions/dbAuth/DbAuthHandler.ts

```diff
@@ -517,10 +528,37 @@ export class DbAuthHandler {
   // pass the argument `expires` set to "now" to get the attributes needed to expire
   // the session, or "future" (or left out completely) to set to `_futureExpiresDate`
   _cookieAttributes({ expires = 'future' }: { expires?: 'now' | 'future' }) {
-    const meta = JSON.parse(JSON.stringify(DbAuthHandler.COOKIE_META))
+    let meta
 
-    if (process.env.NODE_ENV !== 'development') {
-      meta.push('Secure')
+    // DEPRECATED: Remove deprecation logic after a few releases, assume this.options.cookie contains config
+    // TODO: Once old behavior is removed, throw an error if there is no cookie config present
+    if (!this.options.cookie) {
+      console.warn(
+        `\n[Deprecation Notice] dbAuth cookie config has moved to\n  api/src/function/auth.js for better customization.\n  See https://redwoodjs.com/docs/authentication#cookie-config\n`
+      )
+      meta = JSON.parse(JSON.stringify(DbAuthHandler.COOKIE_META))
+
+      if (process.env.NODE_ENV !== 'development') {
+        meta.push('Secure')
+      }
+    } else {
+      meta = Object.keys(this.options.cookie)
+        .map((key) => {
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore-next-line
+          if (this.options.cookie[key] === true) {
```

Comment:
@dac09 Typescript strikes again! I couldn't figure how to get rid of these:

<img width="651" alt="image" src="https://user-images.githubusercontent.com/300/156260002-f593bcfd-3484-40a3-8c79-01c2bc5642a6.png">